### DISCUSSION TITLE
Docker 评测环境问题

### DISCUSSION THREAD

**Discussion by ULis3h** [thu-db/huadb#15] [Q&A] [answered]
想请教个关于Docker评测环境的问题，我拉取了镜像之后docker run hnjylwb/dbtrain 没有任何输出， 直接退出了
运行环境：
macOS Monterey 12.2.
Docker version 27.3.1, build ce12230

**Comment by hnjylwb** [comment, marked as answer]
你需要先使用 `docker run -itd --name dbtrain hnjylwb/dbtrain` 命令启动容器，随后使用 `docker exec -it dbtrain bash` 进入容器环境中

**Comment by ULis3h** [comment]
感谢您的回答，我已经成功启动了这个容器，并且进入了环境中，但是还不会使用，想问一下这个是浏览器访问评测环境吗，还是要把代码拷贝到这个环境里

**Comment by hnjylwb** [comment]
docker 容器的作用是提供一个预安装过软件包的 linux 环境，你可以使用 docker 的 --volume 参数将本地代码映射到 docker 容器中，然后在容器中进行编译和测试。

**Reply by ULis3h** [reply]
明白了， 多谢～
